src/main/java/org/opensearch/security/authtoken/jwt/JwtVendor.java

```diff
@@ -1,171 +0,0 @@
-/*
```

Comment:
Why delete the JWTVendor? Can the token manager use an instance of a JWTVendor to issue tokens for its needs? I think it makes sense to have a generic, reusable JwtVendor.

Reply:
Peter's request: https://github.com/opensearch-project/security/pull/3538#discussion_r1362149896

Reply:
> Can we remove the JwtVendor and replace it with the SecurityTokenManager?

My intention was to removed the JwtVendor from being passed to the Handler and replaced with an instance of the SecurityTokenManager.  That said, I don't mind if we delete the JwtVendor and replace it with the SecurityTokenManager, thats more about naming which I'm indifferent to in this case.

I do have some recommended requirements if you'd like to consider them @scrawfor99:
- Generation and validation for JWTs used by OBO should be the same instance in memory.  There should be only one place where this component is 'new'ed' up, and that single instance is somehow shared everywhere it is needed.
- Dynamic configuration changes can alter the secrets associated with JWTs, this should be allowed.
- The SecurityTokenManager should be able to communicate that it was unable generate a token with an exception.

Reply:
I've made this change